app/views/projects/show.html.erb

```diff
@@ -1,39 +1,113 @@
 <%= render partial: 'shared/iframe_proxy_modal' %>
 
-<h1><%= project.name %></h1>
-
-<script type='text/javascript'>
-  var taxon_data = <%=
-    project
-      .taxons
-      .map { |taxon| { id: taxon.content_id, text: taxon.name } }
-      .to_json
-      .to_s
-      .html_safe
-  %>;
-
-  $(document).ready(function() {
-    $(".select2.tagging_project").select2({
-      allowClear: true,
-      data: taxon_data,
-      multiple: true
-    });
-
-    $(".select2.tagging_project").change(function() {
-      $(this).parents('form').trigger('submit.rails');
-    });
-
-    $("form").on('ajax:success', function(event, data, status, xhr) {
-      $(this).removeClass('error-saving');
-      $(this).effect("highlight", {color: "#d8f3d8"}, 1000);
-    });
-
-    $("form").on('ajax:error', function(event, data, status, xhr) {
-      $(this).addClass('error-saving');
-    });
-  });
-</script>
-
-<div class='tagathon-project-content-list'>
-  <%= render partial: 'content_item', collection: project.content_items.uncompleted %>
+<div class="row">
+  <div class="col-md-3">
+    <h1><%= project.name %></h1>
+
+    <h3>Filter</h3>
+
+    <form method="GET">
+
+      <input id="project-tagging-content-item-search"
+             type="text"
+             class="form-control"
+             name="query"
+             value="<%= params[:query] %>"
+             placeholder="Filter by title...">
+
+      <div class="radio">
+        <label>
+          <input type="radio"
+                 name="taggedState"
+                 id="taggedStateTagged"
+                 value="tagged"
+                 <% if params[:taggedState] == "tagged" %>checked<% end %>
```

Comment:
could set the tagged/not_tagged/all values in a constant somewhere and reference them